examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/emailinput/_cq_template.xml

```diff
@@ -2,4 +2,5 @@
 <jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
     jcr:primaryType="nt:unstructured"
     jcr:title="Email Input"
+          pattern="^(([^&amp;lt;&amp;gt;()\\[\\]\\\\.,;:\\s@&amp;quot;]+(\\.[^&amp;lt;&amp;gt;()\\[\\]\\\\.,;:\\s@&amp;quot;]+)*)|(&amp;quot;.+&amp;quot;))@((([a-zA-Z\\-0-9]+)+[a-zA-Z\\-0-9]{2,})|(\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$"
```

Comment:
Why are we adding the pattern here. We should use the input email and let that form validate the email itself. fieldType=email was already proposed in the form

Reply:
it will validate only if $form.submit was used. how are we taking care of it in runtime? Are we hard cording that pattern in runtime instead of here?

Reply:
On focus out you can call browser API to validate 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/checkValidity

Reply:
This approach was cancelled and we had decided to move on with design dialog instead